mitosheet/src/components/taskpanes/ExcelRangeImport/ExcelRangeImportTaskpane.tsx

```diff
@@ -259,7 +284,94 @@ const ExcelRangeImportTaskpane = (props: ExcelRangeImportTaskpaneProps): JSX.Ele
                                     />
                                 </Col>
                             </Row>
+                            {range_import.type === 'upper left corner value' && 
+                                <>
+                                    <Row justify="space-between">
+                                        <Col>
+                                            <p>
+                                                Table End Condition
+                                            </p>
+                                        </Col>
```

Comment:
To keep the formatting consistent, we could make this a label and tooltip component and add some helpful tooltip

Reply:
I think the question marks might be just additional noise in this case. The dropdown has a ton of information in it. I made the text consistent though.